### PR TITLE
CORTX-33702: Codacy cleanup of markdown files

### PR DESCRIPTION
**Pull request by RohanDhodare**
Signed-off-by: Rohan Dhodare <rohan.v.dhodare@seagate.com>

# Problem Statement
- Codacy warnings in below given files:
  - [fenced-code-flag]
    - doc/FOPFOM-Programming-Guide.md
    - doc/HLD-OF-Motr-LNet-Transport.md
    - doc/HLD-of-Catalogue-Service.md
    - doc/Motr-Epochs-HLD.md
    - doc/Motr-Lnet-Transport.md
    - doc/motr-kv-app.md
    - doc/motr-object-app.md

  - [emphasis-marker]
    - scripts/provisioning/README.md
   
  - [code-block-style]
    - doc/Running_Motr_Across_a_Cluster.md
    - scripts/provisioning/README.md
   
  - [heading-style]
    - doc/Seagate-FDMI-HLD.md

# Design
- [fenced-code-flag] -> added name of programming language after  3 backquotes ```
- [emphasis-marker] -> added * before and after the word to be emphasised
- [code-block-style] -> added 3 backquote ```
- [heading-style] -> used atx-closed formatting for headings

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

-----
[View rendered doc/FOPFOM-Programming-Guide.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/doc/FOPFOM-Programming-Guide.md)
[View rendered doc/HLD-OF-Motr-LNet-Transport.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/doc/HLD-OF-Motr-LNet-Transport.md)
[View rendered doc/HLD-of-Catalogue-Service.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/doc/HLD-of-Catalogue-Service.md)
[View rendered doc/Motr-Epochs-HLD.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/doc/Motr-Epochs-HLD.md)
[View rendered doc/Motr-Lnet-Transport.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/doc/Motr-Lnet-Transport.md)
[View rendered doc/Running_Motr_Across_a_Cluster.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/doc/Running_Motr_Across_a_Cluster.md)
[View rendered doc/Seagate-FDMI-HLD.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/doc/Seagate-FDMI-HLD.md)
[View rendered doc/motr-kv-app.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/doc/motr-kv-app.md)
[View rendered doc/motr-object-app.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/doc/motr-object-app.md)
[View rendered scripts/provisioning/README.md](https://github.com/RohanDhodare/cortx-motr/blob/CORTX-33702/scripts/provisioning/README.md)

### CODE REVIEW

**Comment by rkothiya** [comment]
Can one of the admins verify this patch?

File: doc/Running_Motr_Across_a_Cluster.md [hunk 1]
```diff
@@ -103,10 +103,11 @@ This document provides information on how to build motr from source and then run
 	                 #    pools: [ the pool ]
                 ```
             > Whereas a CDF with 3 nodes should look like this:
-               
+
```
Comment (prasanna46dk) [inline doc/Running_Motr_Across_a_Cluster.md:106]: is this needed? Extra space? inside line

File: doc/Running_Motr_Across_a_Cluster.md [hunk 2]
```diff
@@ -103,10 +103,11 @@ This document provides information on how to build motr from source and then run
 	                 #    pools: [ the pool ]
                 ```
             > Whereas a CDF with 3 nodes should look like this:
-               
+               	
                 # Cluster Description File (CDF).
                 # See `cfgen --help-schema` for the format description.
                 nodes:
+		```sh
```
Comment (prasanna46dk) [inline doc/Running_Motr_Across_a_Cluster.md:110]: This should be yaml/yml, I guess

File: scripts/provisioning/README.md [hunk 3]
```diff
@@ -354,7 +354,9 @@ below for the list of other useful _Vagrant_ commands.
 
 If a cluster-like environment is needed, more machines can be provisioned:
 
-    ./scripts/m0vg up cmu /ssu/ /client/
+```Text
```
Comment (prasanna46dk) [inline scripts/provisioning/README.md:357]: s/Text/sh/
Replace Text with sh as this is shell command(script)

**Review by prasanna46dk — CHANGES_REQUESTED** [review]

File: doc/Running_Motr_Across_a_Cluster.md [hunk 4]
```diff
@@ -101,9 +100,9 @@ This document provides information on how to build motr from source and then run
 	                 #profiles:
```
Comment (prasanna46dk) [inline doc/Running_Motr_Across_a_Cluster.md:100]: Please check tabs setting here same for line # 101, # 102, # 200, # 201, # 202

File: doc/Running_Motr_Across_a_Cluster.md [hunk 5]
```diff
@@ -101,9 +100,9 @@ This document provides information on how to build motr from source and then run
...
 	                 #  - name: default
 	                 #    pools: [ the pool ]
-                ```
+        	```
             > Whereas a CDF with 3 nodes should look like this:
-               
+		    ```yml
```
Comment (prasanna46dk) [inline doc/Running_Motr_Across_a_Cluster.md:105]: check tab

**Review by prasanna46dk — CHANGES_REQUESTED** [review]

File: doc/Running_Motr_Across_a_Cluster.md (hunk 5 above)
Comment (RohanDhodare) [reply doc/Running_Motr_Across_a_Cluster.md:105]: Actually a tab is needed there in order to mark it as code block of particular language

Comment (RohanDhodare) [reply doc/Running_Motr_Across_a_Cluster.md:105]: Had a conversation with @prasanna46dk and understood the problem. will fix it.

File: doc/Running_Motr_Across_a_Cluster.md (hunk 4 above)
Comment (RohanDhodare) [reply doc/Running_Motr_Across_a_Cluster.md]: yes will check and fix it.

**Review by prasanna46dk — APPROVED** [review]

**Comment by prasanna46dk** [comment]
LGTM

**Review by anaghadeshmukh — APPROVED** [review]

**Comment by rkothiya** [comment]
Giving exception to pre-merge and ci/cd as documentation changes

**Pull request merged**